rules/windows/process_creation/proc_creation_win_notepad_local_passwd_discovery.yml

```
@@ -0,0 +1,29 @@
+title: Notepad Password Files Discovery
```

```suggestion
title: Notepad Open Sensitive Password Files
```
The action here isn't about discovery.

---

Please see comment https://github.com/SigmaHQ/sigma/pull/5198#discussion_r1968344964